src/victorops.coffee

```diff
@@ -108,11 +108,12 @@ class VictorOps extends Adapter
     @ws.send( message )
 
   send: (user, strings...) ->
-    js = @chat( strings.join "\n" )
+    #js = @chat( strings.join "\n" )
```

Comment:
Ah, yes. What comes from not reviewing before submission to remove 'test' comments. I'll update.